### ISSUE TITLE
bug (app): Unable to find robots on 3.19.0-alpha.2

### ISSUE THREAD

**Issue by dozercodes** [Opentrons/opentrons#5970]
# Overview

The app fails to locate robots to connect. Instead the app is stuck perpetually looking for robots

# Current behavior

![image](https://user-images.githubusercontent.com/579696/85307147-85fbba00-b47d-11ea-9698-0a029ae44e8a.png)

App fails to find robot to connect to on either Wifi or USB

Mixpanel error stack trace:
```
mixpanel.cjs.js:90 Mixpanel error: Implementation error. Please turn on debug and contact support@mixpanel.com.
j @ mixpanel.cjs.js:90
hostReportError.js:3 Uncaught TypeError: Cannot read property 'opt_out_tracking_cookie_prefix' of undefined
    at pe.get_config (mixpanel.cjs.js:3899)
    at pe._call_gdpr_func (mixpanel.cjs.js:3977)
    at pe.opt_in_tracking (mixpanel.cjs.js:4022)
    at c (mixpanel.js:51)
    at t._tapNext (epic.js:57)
    at t._next (tap.js:45)
    at t.next (Subscriber.js:53)
    at t._next (filter.js:38)
    at t.next (Subscriber.js:53)
    at t._next (pairwise.js:24)
```
# Steps to reproduce

1) Install build 3.19.0-alpha.2 (issue confirmed on windows and mac builds)
2) Open the Opentrons app to the Robot tab
3) Turn on your robot
4) Observe

# Expected behavior

App should find your robot to connect to.

**Comment by dozercodes** [comment]
tagging @sfoster1 for visibility

**Comment by mcous** [comment]
Closed by #5971. For clarity, this was not an issue with robot discovery, but rather, and issue with the app's renderer thread crashing due to an uncaught exception
